src/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -4651,6 +5088,61 @@ impl SimulationElement for SpoilerComputer {
     }
 }
 
+struct A320GravityExtension {
+    gear_gravity_extension_active_id: VariableIdentifier,
+
+    handle_angle: Angle,
+
+    is_extending_gear: bool,
+
+    is_turned: bool,
+}
+impl A320GravityExtension {
+    const INCREMENT_ANGLE_DEGREE_PER_SECOND: f64 = 120.;
+    const ALLOW_RETRACTION_DIRECTION: bool = false;
```

Comment:
You mean the retraction direction?  This one is because we have no animation for the emergency handle, so I prefer to disable ability to revert the emergency extension for now. It can be activated / option removed once we have a way to know where is the emergency handle.

Reply:
Ah yeah, I meant the retraction direction. That makes sense then 👍.

Reply:
Updated name to make this clear